index.html

```diff
@@ -4551,6 +4551,277 @@ <h2>Definition of Roles</h2>
+				</p>
+				<p>
+					To be <a>keyboard accessible</a>, authors SHOULD manage focus of descendants of a <code>listview</code> as described in <a href="#managingfocus">Managing Focus</a>.
+					When users are moving focus among items in a <code>listview</code>, authors SHOULD set focus as follows:
+				</p>
+				<ul>
+					<li>
+						If a <rref>listviewitem</rref> contains a single interactive <rref>widget</rref> that does not consume arrow key presses when it is focus, such as a <rref>checkbox</rref>, <rref>button</rref>, or <rref>link</rref>, focus is set on that contained interactive element instead of on the containing <rref>listviewitem</rref>.
+						This allows the contained widget to be immediately operable and enables assistive technologies to expose the role, name, value, and  state of the contained element.
+					</li>
+					<li>Otherwise, focus is set on a <rref>listviewitem</rref> element.</li>
+				</ul>
+				<p>Authors SHOULD provide a mechanism for moving focus into  and out of a <rref>listviewitem</rref> that allows users to navigate and interact with content contained by the <rref>listviewitem</rref> if the <rref>listviewitem</rref> contains any of the following:</p>
+				<ul>
+					<li>a widget that requires arrow keys to operate, e.g., a <rref>combobox</rref> or <rref>radiogroup</rref></li>
+					<li>multiple interactive elements</li>
+					<li>editable content</li>
+				</ul>
+				<p>
+					Depending on the implementation, pressing <kbd>Enter</kbd>, <kbd>Space</kbd>, or another key would move focus from the <rref>listviewitem</rref> to a focusable element inside the <rref>listviewitem</rref>.
+					Then, pressing <kbd>Enter</kbd> again, <kbd>Tab</kbd>, <kbd>Escape</kbd>, or another key would place focus back on the parent <rref>listviewitem</rref> element so users can continue navigating the <code>listview</code>.
+					For example, if an item in a list of video preview cards contains a video player, the <kbd>Enter</kbd> key might be used to move focus from the <rref>listviewitem</rref> to the play button of the contained player.
+					The card might then  trap focus so users can easily use the <kbd>Tab</kbd> key to navigate among the player's controls and then use the <kbd>Esc</kbd> key or a dynamically rendered close button to return focus to the card.
+				</p>
+				<p>
+					The <code>listview</code> role does not assume any particular form of visual presentation.
+					A <code>listview</code> can scroll in any direction as  users move focus.</code>
+					Items can be arranged in a single column, a single row, or in multiple columns or rows.
+					Authors SHOULD indicate whether the order of the collection is presented horizontally or vertically with <pref>aria-orientation</pref>.
+					For example, if a set of twenty cards is presented in a vertical list that is wrapped into five columns that contain four cards, authors SHOULD set <pref>aria-orientation</pref> to <code>vertical</code>.
```

Comment:
IMO: The value "vertical" would inform screen reader users only that they can use up and down arrow keys to muve through the list. To signal to the users that they can move vertically and horizontally - such as the case in a dashboard - would require a new value for "aria-orientation" in addition to vertical and horizontal. It could be "gridlike", "all" or something similar.

Reply:
My first thought was that this could provide support for languages that are vertical. I wonder if `aria-orientation` could have a default value of `any` (or similar) but could also accept author provided values of `horizontal` or `vertical`?